054-AzureLoadTesting/Coach/README.md

```diff
@@ -0,0 +1,70 @@
+# What The Hack - Azure Load Testing - Coach Guide
+
+## Introduction
+Welcome to the coach's guide for the Azure Load Testing What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.  
+
+NOTE: If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+## Duration
```

Comment:
This belongs under the new template's "Proposed Agenda" section!

Reply:
Done

Reply:
Now that I've re-read this, I think the boilerplate text about needing to package /Student/Resources covers this?  I would either remove your sentence all together, or maybe modify/add a line to the "Student Resources" section in the template.

Reply:
Yeah that makes sense.  I just removed it.